### ISSUE TITLE
Please add LeadCost

### ISSUE THREAD

**Issue by kcaponeAP** [activeprospect/leadconduit-integration-lowermybills#14]
Currently the LMB integration does not support cost of lead. I just received a recent XML from LMB with a new field addition.

Please have this map to the internal LCN field / price

```xml
...<REAgentPhone/>
<Veteran>yes</Veteran>
<LeadCost>5</LeadCost> <-------
<NeedSellingREAgent/>
```

**Comment by ros3bud** [comment]
In progress.

**Comment by ros3bud** [comment]
@kcaponeAP, can you please provide examples of different values they may send?

**Comment by kcaponeAP** [comment]
Do we have an ETA / update on this issue?

Ticket for reference: https://activeprospect.zendesk.com/agent/tickets/6631

**Comment by ros3bud** [comment]
Solved in Version 1.1.4
